api_log/models/api_log_collection.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2025 Simone Rubino - PyTech
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class APILogCollection(models.AbstractModel):
+    _name = "api.log_collection"
```

Comment:
```suggestion
    _name = "api.log_collection.mixin"
```
If I get right your intent.